t.rast.mosaic.py

```diff
@@ -459,14 +459,18 @@ def main():
 
     grass.message(_("Mosaicing the scenes with method %s ...") % options['method'])
     if not options['granularity'] == 'all':
-        grass.run_command('t.rast.aggregate', input=strdsout + '_tmp',
-                          output=strdsout, basename=strdsout,
-                          granularity=options['granularity'],
-                          method=options['method'], quiet=True,
-                          nprocs=options['nprocs'])
+        for method in options['method'].split(","):
+            m_strdsout = strdsout if not "," in options['method'] else f"{strdsout}{method}"
+            grass.run_command('t.rast.aggregate', input=strdsout + '_tmp',
+                              output=strdsout, basename=strdsout,
```

Comment:
```suggestion
                              output=m_strdsout, basename=m_strdsout,
```
different output for each method.